src/components/Patient/ManagePatients.tsx

```
@@ -804,169 +803,116 @@ export const PatientManager = () => {
+              <ExportMenu
+                disabled={!isExportAllowed}
+                exportItems={[
+                  {
+                    label: "Export Live patients",
+                    action: async () => {
+                      const query = {
+                        ...params,
+                        csv: true,
+                        facility: qParams.facility,
+                      };
+                      delete qParams.is_active;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)